ios/brave-ios/Sources/Brave/Frontend/Browser/BrowserViewController/BVC+WKNavigationDelegate.swift

```diff
@@ -97,6 +97,10 @@ extension BrowserViewController: WKNavigationDelegate {
         hideReaderModeBar(animated: false)
       }
     }
+
+    if let tab = tab(for: webView), let url = webView.url, rewards.isEnabled {
```

Comment:
Have we tested against a redirect chain that contains many urls

Reply:
yes, tested with https://www.superuser.com/q/1471861/ url and got following redirect chain:
```
https://www.superuser.com/q/1471861/, 
https://www.superuser.com/questions/1471861/website-with-multiple-redirects,
https://superuser.com/questions/1471861/website-with-multiple-redirects
```

Reply:
Great news. Lets merge and request uplift to beta and stable.